demos/python_demos/whiteboard_inpainting_demo/whiteboard_inpainting_demo.py

```diff
@@ -0,0 +1,198 @@
+    if args.m_instance_segmentation:
+        labels_file = osp.join(root_dir, 'coco_labels.txt')
+        segmentation = MaskRCNN(ie, args.m_instance_segmentation, labels_file,
+                                args.threshold, args.device, args.cpu_extension)
+    elif args.m_semantic_segmentation:
+        labels_file = osp.join(root_dir, 'cityscapes_labels.txt')
+        segmentation = SemanticSegmentation(ie, args.m_semantic_segmentation, labels_file,
+                                            args.threshold, args.device, args.cpu_extension)
+
+    black_board = False
+    output_frame = np.full((frame_size[1], frame_size[0], 3), 255, dtype='uint8')
+    frame_number = 0
+    key = -1
+
+    while True:
+        start = time.time()
+        _, frame = capture.get_frame()
+
+        mask = None
+        if frame is not None:
+            detections = segmentation.get_detections([frame])
+            print(frame_size[0])
+            expand_mask(detections, frame_size[0] // 27)
+            if len(detections[0]) > 0:
+                mask = detections[0][0][2]
+                for i in range(1, len(detections[0])):
+                    mask = cv2.bitwise_or(mask, detections[0][i][2])
+        else:
+            break
+
+        if mask is not None:
+            mask = np.stack([mask, mask, mask], axis=-1)
+        else:
+            mask = np.zeros(frame.shape, dtype='uint8')
+
+        clear_frame = remove_background(frame, invert_colors=not black_board)
+
+        output_frame = np.where(mask, output_frame, clear_frame)
+        merged_frame = np.vstack([frame, output_frame])
+        merged_frame = cv2.resize(merged_frame, out_frame_size)
+
+        if output_video is not None:
+            output_video.write(merged_frame)
+        
+        presenter.drawGraphs(merged_frame)
+        if not args.no_show:
+            cv2.imshow(WINNAME, merged_frame)
+            key = check_pressed_keys(key)
+            if key == 27:  # 'Esc'
+                break
+            if key == ord('i'):  # catch pressing of key 'i'
+                black_board = not black_board
+                output_frame = 255 - output_frame
+            else:
+                presenter.handleKey(key)
+
+        if mouse.crop_available:
+            x0, x1 = min(mouse.points[0][0], mouse.points[1][0]), \
+                     max(mouse.points[0][0], mouse.points[1][0])
+            y0, y1 = min(mouse.points[0][1], mouse.points[1][1]), \
+                     max(mouse.points[0][1], mouse.points[1][1])
+            x1, y1 = min(x1, output_frame.shape[1] - 1), min(y1, output_frame.shape[0] - 1)
+            board = output_frame[y0: y1, x0: x1, :]
+            if board.shape[0] > 0 and board.shape[1] > 0:
+                cv2.namedWindow('Board', cv2.WINDOW_KEEPRATIO)
+                cv2.imshow('Board', board)
+            
+        end = time.time()
+        print('\rProcessing frame: {}, fps = {:.3}' \
+            .format(frame_number, 1. / (end - start)), end="")
```

Comment:
Since there is no line feed printed at the end, at the end of the loop the caret will still be at the end of the text, so any text printed afterwards will be appended to the line. You should fix it by printing a blank line after the loop.

Reply:
Print of empty line added.